test/e2e/app-dir/metadata/metadata.test.ts

```diff
@@ -428,7 +428,7 @@ createNextDescribe(
       it('should pick up opengraph-image and twitter-image as static metadata files', async () => {
         const $ = await next.render$('/opengraph/static')
         expect($('[property="og:image"]').attr('content')).toMatch(
-          /https:\/\/example.com\/_next\/static\/media\/metadata\/opengraph-image.\w+.png/
+          /https:\/\/example.com\/_next\/static\/media\/metadata\/opengraph-image.png/
```

Comment:
Will convert the generation them through app routes later, we can make the image route become `<route>/opengrah-image.png` instead of generating them through webpack. it's easy to match between routes and the parsed image data. And I think the cache will still be handled, so the hash might not be required?

Reply:
Track it in another thread and will bring it back later 🙏